### PR TITLE
Fixes #7056 - only resize textarea in params tab.

### PR DESCRIPTION
**Pull request by tbrisker**
also, take into consideration the padding to prevent unneeded scrolling.

### CODE REVIEW

File: app/assets/javascripts/host_edit.js [hunk 1]
```diff
@@ -622,10 +622,10 @@ function disable_vm_form_fields() {
 }
 
 function resizeTextareaAll () {
```
Comment (orrabin) [inline app/assets/javascripts/host_edit.js]: nitpick, the function name no longer fits the purpose

Comment (tbrisker) [inline app/assets/javascripts/host_edit.js]: accepted, made the function more generic.

**Comment by mmoll** [comment]
works for me :+1:

**Comment by domcleal** [comment]
Thanks @tbrisker, merged as 3b25b6b for Foreman 1.6.0.

**Pull request closed without merging**